services/repository/src/test/java/org/sagebionetworks/repo/web/controller/MetadataProviderFactoryTest.java

```diff
@@ -17,12 +15,13 @@
 import org.sagebionetworks.repo.model.table.MaterializedView;
 import org.sagebionetworks.repo.model.table.SubmissionView;
 import org.sagebionetworks.repo.model.table.VirtualTable;
-import org.sagebionetworks.repo.web.service.metadata.EntityProvider;
-import org.sagebionetworks.repo.web.service.metadata.MetadataProviderFactory;
+import org.sagebionetworks.repo.web.service.metadata.*;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 
+import static org.junit.jupiter.api.Assertions.*;
+
```

Comment:
Looks like your IDE is replacing specific imports with *.  Can you change it to not do that?